mega-linter-runner/lib/runner.js

```diff
@@ -113,13 +113,12 @@ ERROR: Docker engine has not been found on your system.
 
     // Build docker run options
     const lintPath = path.resolve(options.path || ".");
-    const commandArgs = [
-      "run",
-      "-v",
-      "/var/run/docker.sock:/var/run/docker.sock:rw",
-      "-v",
-      `${lintPath}:/tmp/lint:rw`,
-    ];
+    const commandArgs = [ "run" ];
```

Comment:
Nitpick: consider removing whitespace inside square brackets for consistency with other array literals.